### PR TITLE
Change .NET Framework URLs to Internet Archive where possible

### PR DESCRIPTION
**Pull request by wjk**
.NET Framework 1.1 and 2.0SP2 can no longer be downloaded, so I removed their entries instead. [CORE-17555](https://jira.reactos.org/browse/CORE-17555)

### CODE REVIEW

**Review by binarymaster — DISMISSED** [review]
Please revert your removals, because all downloads are available in the Internet Archive.

Links:
```
https://web.archive.org/web/20210505032023if_/http://download.microsoft.com/download/a/a/c/aac39226-8825-44ce-90e3-bf8203e74006/dotnetfx.exe
https://web.archive.org/web/20210212100858if_/http://download.microsoft.com/download/4/f/3/4f3ac857-e063-45d0-9835-83894f20e808/dotnetfx.exe
https://web.archive.org/web/20210126141804if_/http://download.microsoft.com/download/8/f/0/8f023ff4-2dc1-4f10-9618-333f5b9f8040/dotnetfx.exe
https://web.archive.org/web/20210210232422if_/http://download.microsoft.com/download/e/d/a/eda9d4ea-8ec9-4431-8efa-75391fb91421/dotnetfx.exe
https://web.archive.org/web/20200821053624if_/http://download.microsoft.com/download/1/f/a/1fa816d7-a8d6-4f15-b682-b96239e68ab7/dotnetfx.exe
https://web.archive.org/web/20200823074630if_/http://download.microsoft.com/download/c/9/f/c9f672f3-c14b-4cff-9671-d419842d792d/dotnetfx.exe
https://web.archive.org/web/20200821052927if_/http://download.microsoft.com/download/0/8/6/086e7824-ddad-45c0-b765-721e5e28e4c5/dotnetfx.exe
https://web.archive.org/web/20200823070901if_/http://download.microsoft.com/download/a/f/7/af738ebf-dc15-4c61-a20d-1c29306cd9bc/dotnetfx.exe
https://web.archive.org/web/20190720065006if_/http://download.microsoft.com/download/7/b/9/7b90644d-1af0-42b9-b76d-a2770319a568/dotnetfx.exe

https://web.archive.org/web/20190523030127if_/http://download.microsoft.com/download/c/6/e/c6e88215-0178-4c6c-b5f3-158ff77b1f38/NetFx20SP2_x86.exe
```

File: net20.txt [hunk 1]
```diff
@@ -6,7 +6,7 @@ Description = Components needed to run managed Windows applications compiled in
 Size = 22.4 MiB
 Category = 14
 URLSite = http://www.microsoft.com/downloads/details.aspx?FamilyID=0856eacb-4362-4b0d-8edd-aab15c5e04f5
-URLDownload = http://download.microsoft.com/download/5/6/7/567758a3-759e-473e-bf8f-52154438565a/dotnetfx.exe
+URLDownload = https://web.archive.org/web/20210530204812/http://download.microsoft.com/download/5/6/7/567758a3-759e-473e-bf8f-52154438565a/dotnetfx.exe
```
Comment (binarymaster) [inline net20.txt:9]:
```suggestion
URLDownload = https://web.archive.org/web/20210505034809if_/http://download.microsoft.com/download/5/6/7/567758a3-759e-473e-bf8f-52154438565a/dotnetfx.exe
```

File: net40.txt [hunk 2]
```diff
@@ -6,7 +6,7 @@ Description = Components needed to run managed Windows applications compiled in
 Size = 48.1 MiB
 Category = 14
 URLSite = http://www.microsoft.com/en-US/download/details.aspx?id=17718
-URLDownload = https://download.microsoft.com/download/9/5/A/95A9616B-7A37-4AF6-BC36-D6EA96C8DAAE/dotNetFx40_Full_x86_x64.exe
+URLDownload = https://web.archive.org/web/20210530205102/https://download.microsoft.com/download/9/5/A/95A9616B-7A37-4AF6-BC36-D6EA96C8DAAE/dotNetFx40_Full_x86_x64.exe
```
Comment (binarymaster) [inline net40.txt:9]:
```suggestion
URLDownload = https://web.archive.org/web/20210525175036if_/https://download.microsoft.com/download/9/5/A/95A9616B-7A37-4AF6-BC36-D6EA96C8DAAE/dotNetFx40_Full_x86_x64.exe
```

**Review by binarymaster — COMMENTED** [review]
Also, download links should be direct links, not linking to the webarchive html page.

**Comment by wjk** [comment]
@binarymaster Sorry about that; I did not know the files were already archived. Also, for future reference, how do I do a direct download link to an Internet Archive mirrored file? Does the `if_` at the end of the timestamp have anything to do with it?

**Comment by binarymaster** [comment]
> Does the `if_` at the end of the timestamp have anything to do with it?

Yes, that's how it usually works. You can see it's also used in some other files.

**Pull request merged**